README.md

```diff
@@ -54,8 +54,10 @@ wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh |
 
 <sub>The script clones the nvm repository to `~/.nvm` and adds the source line to your profile (`~/.bash_profile`, `~/.zshrc`, `~/.profile`, or `~/.bashrc`).</sub>
 
+<sub>**Note:** If the environment variable `$XDG_CONFIG_HOME` is present, it will place the `nvm` files there.</sub>
```

Comment:
> `$XDG_DATA_HOME` defines the base directory relative to which user specific **data files** should be stored.

> `$XDG_CONFIG_HOME` defines the base directory relative to which user specific **configuration files** should be stored.

Source: https://specifications.freedesktop.org/basedir-spec/basedir-spec-0.6.html

Since `nvm` is a configuration tool that stores different node versions, that could possibly be seen as _data files_. I suppose it's a judgement call.

Reply:
I don't have a strong opinion either way, but I don't want to change it once the decision is made.

Technically I suppose the aliases dir should be "config", the nodes themselves "data", and the cache dir "cache" - but since I'm not going to split up the NVM_DIR, we just have to pick one.

I suppose config is fine, but I thought it was worth discussing. (cc @marcinant)

Reply:
Did I clarify your question or do you need anything more from me?